examples/onepass_mean_var_std.py

```diff
@@ -0,0 +1,84 @@
+"""
+Example: One Pass Mean Var Std
```

Comment:
`Example: One-pass calculation of feature mean, variance, and standard deviation`